packages/v4/patternfly-docs/content/design-guidelines/components/number input/number-input.md

```diff
@@ -40,10 +40,10 @@ Number inputs can function in one of two ways:
 1. **Default:** Buttons increase and decrease the numeric value within a given range.
```

Comment:
Update the "Don't use a number input when" and "Use a number in put when" to "When to use" and "When not to use"

And then replace  "**When using number inputs, follow these best practices:**" with just "best practices" and make them all H3s